app/views/export/index.html.erb

```diff
@@ -1,40 +1,26 @@
 <% content_for :title, 'Export Manager' %>
 
-  <% if @plugin_info.empty? %>
-    <div class="content-container mt-3">
-      <div class="alert alert-error m-0">
-        <p class="lead">Sorry, we were unable to detect any export plugins. This is strange because Dradis ships with a number of plugins installed by default.</p>
-        <p class="mb-0">Check the Export section of your <code><%= Rails.root.join('Gemfile.plugins') %></code> file.</p>
-      </div>
+<% if Dradis::Plugins.with_feature(:export).empty? %>
+  <div class="content-container mt-3">
+    <div class="alert alert-error m-0">
+      <p class="lead">Sorry, we were unable to detect any export plugins. This is strange because Dradis ships with a number of plugins installed by default.</p>
+      <p class="mb-0">Check the Export section of your <code><%= Rails.root.join('Gemfile.plugins') %></code> file.</p>
     </div>
-  <% else %>
+  </div>
+<% else %>
 
-    <ul id="plugin-chooser" class="tabs-container nav nav-tabs">
-      <% @plugin_info.each do |info| %>
+  <ul id="plugin-chooser" class="tabs-container nav nav-tabs">
+    <%= render_view_hooks 'export/plugin_description', feature: :export %>
```

Comment:
I'd like for us to keep naming consistent: for Issues#show calculator tabs we're using:

```erb
<%= render_view_hooks 'issues/show-tabs' %>
```

and 

```
<%= render_view_hooks 'issues/show-content' %>
```

What about using `export/index-tabs` and `export/index-content` here?